devel/irrlichtmt/Portfile

```diff
@@ -18,4 +19,20 @@ checksums           rmd160  4da410c24f67e8c0ebd9880308b8d796d2f86676 \
 
 depends_lib         path:include/turbojpeg.h:libjpeg-turbo \
                     port:libpng \
-                    port:zlib
```

Comment:
Why not keep the dependency on zlib? `libIrrlichtMt.dylib` is still referencing `libz.1.dylib`, which means that zlib should be a direct dependency to ease revision bumps.

Reply:
Did you take the time to read the summary of the commit?

Reply:
1. If mesa becomes a build dependency, the transitive dependency won't work across types of dependencies.
2. When zlib get an ABI breaking change, maintainer of zlib would have to bump revision of all affected dependent ports. If irrlichtmt is not a *direct* dependency of zlib, it could have been missed and we have to fix the breakage ourselves.